scipy/optimize/tnc.py

```diff
@@ -284,7 +290,7 @@ def fmin_tnc(func, x0, fprime=None, args=(), approx_grad=0,
 
 def _minimize_tnc(fun, x0, args=(), jac=None, bounds=None,
                   eps=1e-8, scale=None, offset=None, mesg_num=None,
-                  maxCGit=-1, maxiter=None, eta=-1, stepmx=0, accuracy=0,
+                  maxCGit=-1, maxfun=None, maxiter=None, eta=-1, stepmx=0, accuracy=0,
```

Comment:
When new keywords are added to a function they should be added at the end of the keyword list.

Reply:
I moved it.